cmd/helpgen/main.go

```diff
@@ -38,6 +38,7 @@ const (
 	markerName = "controllertools:marker:generateHelp"
 	header     = `
 // +build !ignore_autogenerated
+// +build !skip
```

Comment:
```suggestion
// +build !cover
```

I am not an expert on this, but IMO the command that motivates this change would look better with another build tag? Compare `go test -cover -tags skip` to `go test -cover -tags cover`. This is only relevant for the test coverage, right?

And why doesn't the existing tag (`ignore_autogenerated`)work,` ref. https://github.com/kubernetes-sigs/kubebuilder/issues/2645#issuecomment-1117057817? Will this just work by adding a new build tag?

Reply:
@Sher-Chowdhury Could you please give your thoughts here.

Reply:
@NikhilSharmaWe, the @erikgb suggestion also makes sense for me.
WDYT about you try to reproduce the issue https://github.com/kubernetes-sigs/kubebuilder/issues/2645#issuecomment-1117057817 and see if you can use the tag `ignore_autogenerated` instead of the skip to sort it out?